cards_corrections.yml

```diff
@@ -1,4 +1,36 @@
 # Objects to be fixed
+#
+# Format is
+#
+# Card Name:
+#   field to fix: new value
+#
+# Ex:
+#
+# Jace, the Planeswalker:
+#   loyalty: 5
+#   manaCost: 1UUB
+#
+# Fields and their values
+# name: Card Name
+# cmc: 4 (int)
+# colorIdentity:
+#   - U
+#   - B
+# colors:
+#   - Blue
+#   - Black
+# manaCost: 1UUB
+# number: 140
+# rarity: Mythic Rare
+# power: X
+# text: "{5}, {T}: You win the game."
+# toughness: *
+# type: Legendary Artifact Creature - Human Monk
+# url: http://wizards.c0m/link/to/card.png
```

Comment:
p/t is not a generated value, so we can/should correct it in `cards_corrections` manually!

The potential subpage scraping and result caching is a different story.
Maybe a very sweet option for the future to get better data more automated. mtgs updates and corrects their data on the page from time to time.